docs/pages/docs/migration-guide.mdx

```diff
@@ -7,6 +7,83 @@ import { FileTree, Steps, Tabs, Callout } from "nextra/components";
 
 # Migration guide
 
+## 0.9.0-next.0
+
+### API functions
+
+API functions are required and Ponder will fail to start if `src/api/index.ts` does not exist.
+
+Copy the following code into `src/api/index.ts` to achieve the same functionality as `<=0.8`.
+
+```ts filename="src/api/index.ts"
+import { db } from "ponder:api";
+import schema from "ponder:schema";
+import { Hono } from "hono";
+import { graphql } from "ponder";
+
+const app = new Hono();
+
+app.use("/graphql", graphql({ db, schema }));
+
+export default app;
+```
+
+#### Removed `ponder.get()`
```

Comment:
Removed `ponder.get()`, `post()` and `use()`

Now, simply use Hono's built-in routing system.